masonite/commands/ModelCommand.py

```diff
@@ -1,15 +1,45 @@
 """New Model Command."""
-from masonite.commands import BaseScaffoldCommand
+import subprocess
 
+from masonite.app import App
+from masonite.helpers.filesystem import make_directory
+from masonite.view import View
 
-class ModelCommand(BaseScaffoldCommand):
+from cleo import Command
+
+
+class ModelCommand(Command):
     """
     Creates a model.
 
     model
         {name : Name of the model}
+        {--m|--migration : Create a migration for specified model}
     """
 
     scaffold_name = "Model"
     template = '/masonite/snippets/scaffold/model'
     base_directory = 'app/'
+
+    def handle(self):
+        class_name = self.argument('name')
+        view = View(App())
+        class_directory = '{}{}.py'.format(self.base_directory, class_name)
+
+        if not make_directory(class_directory):
+            return self.error('{0} Already Exists!'.format(self.scaffold_name))
+
+        f = open(class_directory, 'w+')
+        if view.exists(self.template):
+            f.write(
+                view.render(self.template, {
+                            'class': class_name.split('/')[-1]}).rendered_template
+            )
+            self.info('{} Created Successfully!'.format(self.scaffold_name))
+            f.close()
+
+        if self.option('migration'):
+            model_name = class_name.lower() + 's'
+            subprocess.call(['orator', 'make:migration', 'create_{}_table'.format(model_name),
```

Comment:
I tried doing `self.call` but it was giving a weird error, `[TypeError]
  can only concatenate list (not "str") to list` that I wasn't able to debug, that is why I used subprocess.